specification/network/resource-manager/Microsoft.Network/stable/2023-11-01/virtualWan.json

```diff
@@ -9713,6 +9943,38 @@
         "name": "VnetLocalRouteOverrideCriteria",
         "modelAsString": true
       }
+    },
+    "ConnectionSharedKey": {
+      "properties": {
+        "value": {
+          "type": "string",
```

Comment:
This looks like a secret value, if yes, please use [https://github.com/Azure/autorest/blob/main/docs/extensions/readme.md#x-ms-secret](https://github.com/Azure/autorest/blob/main/docs/extensions/readme.md#x-ms-secret) here.

Reply:
Fixed it